tests/test_matrices.py

```diff
@@ -74,14 +74,14 @@ def test_is_symmetric(self):
         sym_mat = gs.array([[1., 2.],
                             [2., 1.]])
         result = self.space.is_symmetric(sym_mat)
-        expected = gs.array([[True]])
+        expected = gs.array(True)
```

Comment:
These weird shapes were enforced in order to have vectorized functions. Since you are changing them, could you add 1 or 2 tests to see if your functions are vectorized? (they are probably, as they rely on numpy). Thanks!

Reply:
done :)